src/fragment-components/a-radiobutton-group.tsx

```diff
@@ -0,0 +1,204 @@
+			titleText='Default selection'
+			items={allItems}
+			initialSelectedItem={allItems.find((item: any) => item.defaultChecked)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				defaultSelected: `${event.selectedItem.id}`,
+				valueSelected:`${event.selectedItem.id}`,
+				items: selectedComponent.items.map((item: any) => ({
+						...item, 
+						defaultChecked: event.selectedItem.id === item.id ? true : false
+				}))
+		})}/>
+	</>
+};
+
+
+export const ARadioButtonGroup = ({
+	children,
+	componentObj,
+	...rest
+}: any) => {
+return (
+		<AComponent
+		componentObj={componentObj}
+		{...rest}>
+			<RadioButtonGroup
+			className={componentObj.cssClasses?.map((cc: any) => cc.id).join(' ')}
+			legendText= {componentObj.legend}
+			disabled= {componentObj.disabled}
+			orientation={componentObj.orientation}
+			defaultSelected={componentObj.defaultSelected}
+			valueSelected={componentObj.valueSelected}
+			labelPosition={componentObj.labelPosition}
+			name={componentObj.codeContext?.name}>
+					{children}
+			</RadioButtonGroup>
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: ARadioButtonGroup,
+	styleUI: ARadioButtonGroupStyleUI,
+	codeUI: ARadioButtonGroupCodeUI,
+	keywords: ['radio', 'button', 'group'],
+	name: 'Radio buttons',
+	defaultComponentObj: {
+		disabled: false,
+		type: 'radioButtonGroup',
+		legend: 'Radio Button Group',
+		defaultSelected: '',
+		valueSelected: '',
+		labelPosition: 'left',
+		orientation: 'horizontal',
+		items: [
+			{
+				type: 'radioButton',
+				labelText: "Option 1",
+				disabled: false,
+				defaultChecked: true,
+			},
+			{
+				type: 'radioButton',
+				labelText: "Option 2",
+				disabled: false,
+				defaultChecked: false,
+			},
+			{
+				type: 'radioButton',
+				labelText: "Option 3",
+				disabled: false,
+				defaultChecked: false,
+			}
+		]
+	},
+	render: ({ componentObj, select, remove, selected, renderComponents }) => <ARadioButtonGroup
+		componentObj={componentObj}
+		select={select}
+		remove={remove}
+		selected={selected}>
+			{componentObj.items.map((button: any) => (renderComponents(button)))}
+	</ARadioButtonGroup>,
+	image,
+	codeExport: {
+		angular: {
+			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}LegendText = "${json.legend}";
+								@Input() ${nameStringToVariableString(json.codeContext?.name)}Orientation = "${json.orientation}";
+								@Input() ${nameStringToVariableString(json.codeContext?.name)}LabelPosition = "${json.labelPosition}";
+								@Input() ${nameStringToVariableString(json.codeContext?.name)}Name = "${json.codeContext?.name}";
+								@Input() ${nameStringToVariableString(json.codeContext?.name)}defaultSelected = "${json.defaultSelected}";
+								@Input() ${nameStringToVariableString(json.codeContext?.name)}valueSelected = "${json.valueSelected}";`,
+			outputs: (_) => '',
```

Comment:
Pretty sure there is an output event 🤔

Reply:
i dont think so.. currently the states get updated correctly